proto/beacon/p2p/v1/messages.pb.go

```diff
@@ -5,12 +5,11 @@ package ethereum_beacon_p2p_v1
 
 import (
 	fmt "fmt"
-	io "io"
-	math "math"
-
 	proto "github.com/gogo/protobuf/proto"
 	types "github.com/gogo/protobuf/types"
 	v1alpha1 "github.com/prysmaticlabs/prysm/proto/eth/v1alpha1"
+	io "io"
+	math "math"
```

Comment:
why does this happen, do you wanna revert this?

Reply:
pbs generation did not got fix by goimport

Reply:
reverted